src/interfaces/SortType.ts

```diff
@@ -0,0 +1,4 @@
+export enum SortType {
+  name = 'name',
+  length = 'length',
+}
```

Comment:
you should add default value into SortType

`Use enum SortType instead of string literals for possible sorting options (including the default one)`